client/layout/masterbar/checkout.tsx

```diff
@@ -91,7 +93,7 @@ const CheckoutMasterbar = ( {
 			} ) }
 		>
 			<div className="masterbar__secure-checkout">
-				{ showCloseButton && (
+				{ showCloseButton && ! shouldUseCheckoutV2 && (
```

Comment:
I don't agree that it's a good idea to lock users into checkout with no way out. That was the whole reason we added this button in the first place. I think we should push back on this feedback request.

Reply:
I do agree here. I wonder if changing the WordPress.com logo into a 'back' button would be better? I think just a general 'exit' point would be most important regardless of the method used. I believe Jetpack already removed their back button in production 🤔 

Either way, my gut feeling is I don't like being 'locked in' to a payment page since I'm accustomed to having full control of page navigation. I know that I have my browser back button but it still makes me search around frantically for the checkout page back button.

Reply:
Added a comment to the o.g. CFT post pdtkmj-2p7-p2#comment-4529